api/mesh/v1alpha1/mesh.proto

```diff
@@ -123,6 +123,14 @@ message TracingBackend {
   google.protobuf.Struct conf = 4;
 }
 
+message DatadogTracingBackendConfig {
+  // IP address of datadog collector.
```

Comment:
Is this required to be an IP address, or can a host name work too? IIUC, using a DNS cluster should be ok for either.

Reply:
I'll test using a hostname here